asserts/system_user.go

```diff
@@ -216,6 +235,18 @@ func checkHashedPassword(headers map[string]interface{}, name string) (string, e
 	return pw, nil
 }
 
+func checkSystemUserExpiration(headers map[string]interface{}) (string, error) {
+	str, err := checkOptionalString(headers, "user-valid-for")
```

Comment:
I think to have more forward flexibility if we are asked to expand features, I think we want to call this field either:

```
user-presence: until-expiration
```

the presence term is already used for snaps for example to constraint installation state.

Or otherwise maybe:

```
user-lifetime: until-expiration
```

@degville any thoughts on these terminology options or other suggestions?

Reply:
I don't know if I'm being a little over-sensitive, but `user-lifetime: until-expiration` makes me think of the user physically dying! For that reason, I'd go with _user-presence_.

Reply:
thanks, that is what I wondered about myself and why I asked your input. If you are OK with it, I think we should go with `user-presence`, as in "presence on the system".

Reply:
Thank you so much for all the input here guys! I'll get it updated today!